### DISCUSSION TITLE
SSL证书异常

### DISCUSSION THREAD

**Discussion by fxln0** [1Panel-dev/1Panel#5056] [Q&A]
### 1Panel 版本

1.10.7-lts

### 问题描述

![image](https://github.com/1Panel-dev/1Panel/assets/168253884/1a295cce-82ff-453c-9c73-246b2764381a)
![image](https://github.com/1Panel-dev/1Panel/assets/168253884/d037c575-1b56-460e-ab5d-f9b36d896dd2)
![image](https://github.com/1Panel-dev/1Panel/assets/168253884/bfbf656c-c2be-476c-9649-8ffcb4dae918)

申请SSL证书成功后异常，证书里没有任何内容

### 重现步骤

申请ZeroSSL完成后就这样了

**Comment by zhengkunwang223** [comment]
重新申请之后也没有吗？

**Comment by fxln0** [comment]
是的，有时候也会申请失败

**Comment by fxln0** [comment]
![83f19d86acd8657a2a4e8f330f36cbb](https://github.com/1Panel-dev/1Panel/assets/168253884/782c4d87-072d-4ac0-9e85-e0b54a1d727b)
![9b42e5d4e7ee64effbe375b81c35b04](https://github.com/1Panel-dev/1Panel/assets/168253884/6629cd84-e737-4f74-8a75-4eb825febfe9)
有的时候连账号都申请不了

**Comment by fxln0** [comment]
重装了系统也是这样，外网是可以正常访问的

**Comment by zhengkunwang223** [comment]
那就是网络问题了 看你的错误都是超时和主机不可达 你的机器不能正常访问证书签发机构
